Makefile

```diff
@@ -123,18 +128,6 @@ ifndef LINT_LANG
 	LINT_LANG="all"
 endif
 
-ifeq ($(USE_MKLDNN), 1)
-	CFLAGS += -DMXNET_USE_MKLDNN=1
-	CFLAGS += -DUSE_MKL=1
-	CFLAGS += -I$(ROOTDIR)/src/operator/nn/mkldnn/
-	ifneq ($(MKLDNNROOT), $(MKLROOT))
```

Comment:
Previously we have this check because `MKLROOT` might be changed when build mshadow.